### PR TITLE
document the hardware

### LINKED ISSUE #12: Hardware documentation missing

### PR DESCRIPTION
**Pull request by ssisbit**
How did the hardware evolve, what is it build by?

### CODE REVIEW

File: hardware/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ssis:bit hardware
+
+The foundation of the ssis:bit is a [LilyGo T8 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) board with [CircuitPython](https://circuitpython.org/). It has already many key features out of the box:
+
+- ESP32-S2 with 240 MHz CPU, 4GB Flash, 8GB PSRAM
+- 1.14" ST7789 Display 240x135
```
Comment (ssis-robotics) [inline hardware/README.md:6]: In REPL it is actually 37x9 characters (333). Less than the 2K from 80x25 of old terminals, but usable nonetheless.

File: hardware/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- WIFI4 2.4 GHz
+- microSD card slot for sensor data or images
```
Comment (ssis-robotics) [inline hardware/README.md:8]: How to access from CircuitPython?

File: hardware/README.md (hunk 1 above)
Comment (ssis-robotics) [inline hardware/README.md:6]: Please draw at least one Mandelbrot

**Review by ghost — COMMENTED** [review]
Well expanded, thanks for the picture.

**Review by kreier — COMMENTED** [review]
Picture should be included in the main page as well.

**Pull request merged**